easybuild/easyconfigs/l/libcifpp/libcifpp-7.0.8-GCC-13.3.0.eb

```diff
@@ -0,0 +1,43 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2025/01
+easyblock = 'CMakeMake'
+
+name = 'libcifpp'
+version = '7.0.8'
+
+homepage = 'https://github.com/PDB-REDO/libcifpp'
+description = """This library contains code to work with mmCIF and PDB files"""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+github_account = 'PDB-REDO'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['2297e6649a4f71caf9da5f1d97f59512e7324bb62083bb5b08eb00c1c0385cb3']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('Eigen', '3.4.0'),
+    ('Boost', '1.85.0'),  # req. for tests
+]
+
+dependencies = [
+    ('zlib', '1.3.1'),
+    ('mrc', '1.3.13'),
+]
+
+configopts = [
```

Comment:
Something is getting downloaded during configuration, we try hard to make sure that doesn't happen.
Can you try to fix that?

Reply:
seems that I lost some of the previous changes. There used to be a configopt -DCIFFPP_DOWNLOAD_CCD=OFF.
I'll have a closer look tomorrow.